### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'line'

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#377]
$make linkcheck

```
Exception occurred:
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/builders/linkcheck.py", line 70, in check
    lineno = node.line
AttributeError: 'NoneType' object has no attribute 'line'

# Sphinx version: 0.6.5
# Docutils version: 0.5 release
Traceback (most recent call last):
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/cmdline.py", line 170, in main
    app.build(all_files, filenames)
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/application.py", line 136, in build
    self.builder.build_update()
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/builders/__init__.py", line 265, in build_update
    'out of date' &#37; len(to_build))
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/builders/__init__.py", line 320, in build
    self.write(docnames, list(updated_docnames), method)
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/builders/__init__.py", line 359, in write
    self.write_doc(docname, doctree)
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/builders/linkcheck.py", line 54, in write_doc
    self.check(node, docname)
  File "/Library/Python/2.5/site-packages/Sphinx-0.6.5-py2.5.egg/sphinx/builders/linkcheck.py", line 70, in check
    lineno = node.line
AttributeError: 'NoneType' object has no attribute 'line'
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/377
- Originally reported by: Anonymous
- Originally created at: 2010-04-06T20:53:51.714

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-04-18 07:56:50+00:00_

Thanks, should be fixed in <<cset f5d52fa4f454db08b6acb661748d1811770a959e>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:17+00:00_

Removing version: 0.6.4 (automated comment)
